utils/resultstable.go

```diff
@@ -255,13 +255,36 @@ func prepareVulnerabilities(vulnerabilities []services.Vulnerability, results *R
 
 func sortVulnerabilityOrViolationRows(rows []formats.VulnerabilityOrViolationRow) {
 	sort.Slice(rows, func(i, j int) bool {
-		if rows[i].SeverityNumValue != rows[j].SeverityNumValue {
+		switch {
+		case rows[i].SeverityNumValue != rows[j].SeverityNumValue:
 			return rows[i].SeverityNumValue > rows[j].SeverityNumValue
+		case rows[i].Applicable != rows[j].Applicable:
+			return convertApplicableToScore(rows[i].Applicable) > convertApplicableToScore(rows[j].Applicable)
+		case rows[i].JfrogResearchInformation != nil || rows[j].JfrogResearchInformation != nil:
+			ok, value := JfrogResearchInformationSort(rows[i].JfrogResearchInformation, rows[j].JfrogResearchInformation)
+			if ok {
+				return value
+			}
+			return rows[i].IssueId > rows[j].IssueId
+		default:
+			return rows[i].IssueId > rows[j].IssueId
 		}
-		return len(rows[i].FixedVersions) > 0 && len(rows[j].FixedVersions) > 0
 	})
 }
 
+func JfrogResearchInformationSort(jfrogResearchInformation1 *formats.JfrogResearchInformation, jfrogResearchInformation2 *formats.JfrogResearchInformation) (bool, bool) {
+	switch {
+	case jfrogResearchInformation1 == nil:
+		return true, true
+	case jfrogResearchInformation2 == nil:
+		return true, false
+	case jfrogResearchInformation1.SeverityNumValue != jfrogResearchInformation2.SeverityNumValue:
+		return true, jfrogResearchInformation1.SeverityNumValue > jfrogResearchInformation2.SeverityNumValue
+	default:
+		return false, false
+	}
+}
```

Comment:
* Rename for readability (suggestion: `CompareJfrogResearchInformation`). 
* add a description to the func, its not very easy to guess what is the output values. why do we need 2 values? usually. for comparison, the output has only one int value (0 == equal, 0> , 0<)
* What happened if both are nil?